wrapper/ibus/setup/main.py

```diff
@@ -1,3 +1,5 @@
+#!/usr/bin/python3
```

Comment:
For the wrapper it's probably better to use `#!/usr/bin/env python3` instead, see https://github.com/sunpinyin/sunpinyin/pull/113